integration_tests/test_math_03.py

```diff
@@ -1,4 +1,5 @@
-from math import (cbrt, exp2)
+from math import cbrt
+from numpy import exp2

```

Comment:
It seems CPython 3.11 supports exp2, see: https://github.com/lcompilers/lpython/pull/1594

You can add new tests for exp and exp2 in this or a new file.